api/src/main/java/org/phenotips/mendelianSearch/internal/PatientViewUtils.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.phenotips.mendelianSearch.internal;
+
+import org.phenotips.mendelianSearch.PatientView;
+
+import org.xwiki.component.annotation.Component;
+
+import java.util.Comparator;
+import java.util.List;
+
+import javax.inject.Singleton;
+
+import net.sf.json.JSONObject;
+
+/**
+ * @Version $Id$
+ */
+
+@Component(roles = { PatientViewUtils.class })
```

Comment:
If this doesn't need any other components injected, then it doesn't have to be a component at all, just use it as a static utility class.